### PR TITLE
Fix login button coming over other components

### PR DESCRIPTION
**Pull request by maaverik**
Fixed url change, looking at login button blocking copy button on firefox

### CODE REVIEW

File: simplq/src/styles/navbar.module.scss [hunk 1]
```diff
@@ -65,9 +65,11 @@
     }
 
     // In case of smaller screens with sidebar, display at the bottom of sidebar
-    bottom: 1rem;
+    @media (max-width: 787px) {
+      bottom: 1rem;
+    }
     @media (min-width: 787px) {
```
Comment (maaverik) [inline simplq/src/styles/navbar.module.scss:71]: Was blocking buttons on firefox, but strangely not on chrome. Have to figure out why these certain styles act differently.

Screenshot from 2020-10-28 01-20-40

Comment (daltonfury42) [reply simplq/src/styles/navbar.module.scss:71]: I fixed it like [this](https://github.com/SimplQ/simplQ-frontend/pull/396/files#diff-e18458cc3490415771c0eabd0f4970247727d4e31c941003955a4da4aa8a5c91R62)... can you pull latest code?

File: simplq/src/components/pages/Admin/ShareQueue.jsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { ShareButton } from '../../common/Button/Button.stories';
 
 const ShareQueue = (props) => {
   const [clicked, setClicked] = useState(false);
-  const shareUrl = `Hi! Use ${window.location.origin}/j/${props.queueName} to join my queue and get live updates.`;
```
Comment (daltonfury42) [inline simplq/src/components/pages/Admin/ShareQueue.jsx:8]: Me and @sajmalyousef made this change yesterday. 

Rationale was that people would copy like and paste it on some messenger or something, and this text would be a good idea.

File: simplq/src/styles/navbar.module.scss (hunk 1 above)
Comment (daltonfury42) [reply simplq/src/styles/navbar.module.scss:71]: It's happening on chrome too

Comment (maaverik) [reply simplq/src/styles/navbar.module.scss:71]: The height: min-content didn't fix it on firefox on me. I guess that fixed it on chrome though.

File: simplq/src/components/pages/Admin/ShareQueue.jsx (hunk 2 above)
Comment (maaverik) [reply simplq/src/components/pages/Admin/ShareQueue.jsx:8]: Okay, that makes sense.

File: simplq/src/styles/navbar.module.scss (hunk 1 above)
Comment (daltonfury42) [reply simplq/src/styles/navbar.module.scss:71]: nice. The bottom 1rem was causing this. Interesting find.

**Review by daltonfury42 — APPROVED** [review]

**Pull request merged**